lib/generators/graphiti/generator_mixin.rb

```diff
@@ -25,6 +25,33 @@ def api_namespace
       end
     end
 
+    def namespace_controllers?
+      @namespace_controllers ||= begin
```

Comment:
@LoganDSPrice I've never seen the `||=` operator used with `begin`, what exactly is happening here?

Reply:
I followed the style of their `api_namespace` method. I think the `begin` just lets you write a block of code and the return of that block is what the `||=` will be set to if @namespace_controllers isn't set.

Reply:
`@namespace_controllers` being set to whatever `graphiti_config["namespace-controllers"]` is, if it doesn't already exist. `@namespace_controllers` evaluates to a `Boolean` then? Is there a reason why it's an instance variable? I don't see it used anywhere else. @LoganDSPrice